php/src/Snagshout/Promote/Resource/UsersResource.php

```diff
@@ -79,4 +79,35 @@ public function checkEmail(\Snagshout\Promote\Model\CheckEmailRequestBody $body,
 
         return $response;
     }
+
+    /**
+     *
+     *
+     * @param \Snagshout\Promote\Model\CheckEmailRequestBody $body
+     * @param array  $parameters List of parameters
+     * @param string $fetch      Fetch mode (object or response)
+     *
+     * @return \Psr\Http\Message\ResponseInterface|\Snagshout\Promote\Model\Error
+     */
+    public function requestPayout(\Snagshout\Promote\Model\RequestPayout $body, $parameters = [], $fetch = self::FETCH_OBJECT)
+    {
```

Comment:
At the very top add an if statement and if $email is not set in $parameters set throw an exception

Reply:
done